### ISSUE TITLE
`schema` should support absolute paths

### ISSUE THREAD

**Issue by ankon** [sid88in/serverless-appsync-plugin#382]
https://github.com/sid88in/serverless-appsync-plugin/blob/18644ea4b11fc96105b88c3d075a8864ba1bc5c7/src/get-config.js#L61-L67

This (somewhat explicitly in the name) assumes that the `config.schema` would be a relative path, but that is not documented.

1. Is there a particular reason for this requirement, or could this be relaxed?
   ```diff
   diff --git a/src/get-config.js b/src/get-config.js
   index 597e084..17e1764 100644
   --- a/src/get-config.js
   +++ b/src/get-config.js
   @@ -59,7 +59,7 @@ const getConfig = (config, provider, servicePath) => {
      const mappingTemplates = config.mappingTemplates || [];
    
      const readSchemaFile =
   -      schemaRelPath => fs.readFileSync(path.join(servicePath, schemaRelPath), { encoding: 'utf8' });
   +      schemaPath => fs.readFileSync(path.resolve(servicePath, schemaPath), { encoding: 'utf8' });
    
      const schemaContent =
        Array.isArray(config.schema) ?
   ```    
2. If there is a reason, would you accept a PR to clarify the documentation?

**Comment by bboure** [comment]
@ankon Yes!. That sounds good. 
Usually, the schema is in the same or in a subdirectory of the repository, but I don't see any issue with using Absolute paths.

Feel free to open a PR.

What is your use-case?

In addition to that, I have recently found myself in need of using something like glob expressions, in order to split the schema into several files more easily.

The plugin already supports schema stitching, but it requires specifying all the files one by one.

Something like would be a great improvement:

```yml
schema: schemas/*.graphql
```

Since it touches the same part of the code, is that something you'd like to help with too?

Thanks!!

**Comment by ankon** [comment]
> What is your use-case?

I'm trying to build a serverless plugin that auto-generates part of the configuration for our specific environment. Having such "arbitrary" restrictions is "annoying" (easy to work around, but hitting them always has this "Doh!" feeling)

> [globbing]

Interesting one. I probably would consider it part of a different PR/issue, but it doesn't sound particularly hard to do (Famous last words ...)

**Comment by sid88in** [comment]
Good catch — and it survived into v2. The schema paths were being resolved with
`path.join(servicePath, schema)`, which mangles absolute paths (it prefixes them with the
service dir), so an absolute `schema` path quietly resolved to nothing.

Fixed in #721  by switching to `path.resolve` — exactly the change you suggested. Relative
paths still resolve against the service directory; absolute paths are now honored. Added
tests for both.

(The glob part of this issue — `schema: schemas/*.graphql` — is already supported on v2, so
#NNN just closes the absolute-path gap.)

Thanks @ankon!
